ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -48,7 +48,7 @@ Requirements are located here: [avd-requirements](../../README.md#Requirements)
 
```

Comment:
Comments for "Role Inputs and Outputs":
- Under "Inputs": "Structured EOS configuration according to supported data models"
- Under "Tasks": "1. Import device structured configuration from YAML file unless `structured_config` flag is set by `eos_designs`